test/integration/data_repo_client_test.rb

```diff
@@ -18,8 +18,8 @@ class DataRepoClientTest < ActiveSupport::TestCase
 
   # skip a test if the TDR API is not up (since it is their dev instance there is no uptime guarantee)
   def skip_if_api_down
-    if !@data_repo_client.api_available?
-      puts "-- skipping due to TDR API being unavailable --" ; skip
+    unless @data_repo_client.api_available?
```

Comment:
Ironically, even though it's plain English, I find `unless` harder to understand than `if !`.  I'll adapt!